### PR TITLE
OLS-261: Refactor SUID generator

### PR DESCRIPTION
**Pull request by tisnik**
## Description

- Refactor SUID generator
- We used to have three different `utils` which is too heavy for such a small project. That one utils can be refactored w/o problems.

## Type of change

- [x] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library

## Related Tickets & Documents
Related Issue #[OLS-261](https://issues.redhat.com//browse/OLS-261)

### CODE REVIEW

File: ols/utils/suid.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+import uuid
+
+
+def get_suid() -> str:
+    """Generate a unique session ID (SUID) using UUID4.
+
+    Returns:
+        A unique session ID.
+    """
+    return str(uuid.uuid4().hex)
```
Comment (onmete) [inline ols/utils/suid.py:12]: Just to be sure, we don't mind that uuid is not "partitioned" by `-`?
image

Comment (tisnik) [reply ols/utils/suid.py:12]: we actually expect the 1st format

Comment (onmete) [reply ols/utils/suid.py:12]: UUID without hyphens is still technically a valid UUID, but its format deviates from the standard and widely recognized hyphenated notation.
So what is the reason that we expect it without hyphens?

Comment (tisnik) [reply ols/utils/suid.py:12]: - You need to ask original author Vishna
- Do you expect this change to be part of refactoring?
- Where do we get the info, that we must use UUID? We just need some session ID, that's it. Can be generated by other means

Comment (onmete) [reply ols/utils/suid.py:12]: It is outside of this PR scope. We'll resolve this offline.

**Pull request merged**